docs/resources/question.md

```diff
@@ -39,6 +39,7 @@ resource "jupiterone_question" "unencrypted_critical_data_stores" {
 - `compliance` (Block List) (see [below for nested schema](#nestedblock--compliance))
 - `polling_interval` (String) Frequency of automated question evaluation. Defaults to ONE_DAY.
 - `query` (Block List) (see [below for nested schema](#nestedblock--query))
+- `show_trend` (Boolean) Whether to enable daily trend data collection. Defaults to false.
```

Comment:
Maybe just update "daily trend data" to trend data, because its dependant on the polling interval and won't always be daily.